library/ui/restrict-extent/README.md

```diff
@@ -0,0 +1,3 @@
+## Resctrict area
+
+Script limits an area for a user to go around a map. For example if you have data only in a certain country, you can limit an area of a map to a certain predefined in QGIS extent like at the video below.
```

Comment:
Not sure about which video you are referring too.

Reply:
Oh yeah my bad. That was a copy from some other README. I'll change it right away